internal/pfcp/pfcp.go

```diff
@@ -75,6 +76,7 @@ func NewPfcpServer(cfg *factory.Config, driver forwarder.Driver) *PfcpServer {
 		txTrans:      make(map[string]*TxTransaction),
 		rxTrans:      make(map[string]*RxTransaction),
 		log:          logger.PfcpLog.WithField(logger.FieldListenAddr, listen),
+		urrURSEQN:    make(map[uint32]uint32),
```

Comment:
what is `urrURSEQN` used for? It is not used in this PR. If no more use, please remove it.